src/modules/adsupport.js

```diff
@@ -3,7 +3,7 @@ export default function (playerInstance, options) {
     const VPAID_VERSION = '2.0';
 
     playerInstance.renderLinearAd = (adListId, backupTheVideoTime) => {
-        playerInstance.toggleLoader(true);
+        playerInstance.toggleLoader(false);
```

Comment:
Won't this have some undesired impacts? Like not showing the loader while ad is being loaded? Pretty sure the loader there is intentional..

Reply:
@golikov With this, the loader doesn't appear at the same time as the play button. I've tested it with the slow net and I see the loader popping up when it's loading more content

Reply:
I am not going to be testing that just to prove a point, I still don't believe that won't break a loder where we kind of expect it, so let's hope the QA catches that.